docs/faq.md

```diff
@@ -31,16 +31,7 @@ https://github.com/docker/docker/blob/master/LICENSE)
 
 ### Does Docker run on Mac OS X or Windows?
 
-Docker Engine currently runs only on Linux, but you can use VirtualBox to run
-Engine in a virtual machine on your box, and get the best of both worlds. Check
```

Comment:
VMs, esp. OSS VMs like VirtualBox hosting an installed linux distro are still going to be a very popular way to run Docker Engine. Should probably keep most of this text. No?